CFC_DataCollector/recommender/recommendation_pipeline.py

```diff
@@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
         return []
 
     def get_selected_user_utility_model(self, user_id):
-        return []
+        return UserUtilityModel.find_from_db(user_id)
 
     def recommend_trips(self, trip_id, utility_model):
         return []
```

Comment:
I did. Today morning. I forwarded his email so maybe it looks like it came
from him?

Shankari
On Apr 29, 2015 5:48 PM, "Shaun Singh" <notifications@github.com> wrote:

> In CFC_DataCollector/recommender/recommendation_pipeline.py
> <https://github.com/e-mission/e-mission-server/pull/111#discussion_r29396093>
> :
>
> > @@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
> >          return []
> >
> >      def get_selected_user_utility_model(self, user_id):
> > -        return []
> > +        return UserUtilityModel.find_from_db(user_id)
> >
> >      def recommend_trips(self, trip_id, utility_model):
> >          return []
>
> Ok, thanks. Multiplying probabilities doesn't quite make sense with our
> model... Can you send me vijs? Thanks. As far as production, early next
> week (Monday?) should be fine, especially if other students are available
> to help.
> … <#14d07cb9ccccfc54_>
> On Wednesday, April 29, 2015, shankari <notifications@github.com> wrote:
> In CFC_DataCollector/recommender/recommendation_pipeline.py <
> https://github.com/e-mission/e-mission-server/pull/111#discussion_r29350629>
> : > @@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list): >
> return [] > > def get_selected_user_utility_model(self, user_id): > -
> return [] > + return UserUtilityModel.find_from_db(user_id) > > def
> recommend_trips(self, trip_id, utility_model): > return [] Let me send you
> Vij's code and we can see how easy it is to use. — Reply to this email
> directly or view it on GitHub <
> https://github.com/e-mission/e-mission-server/pull/111/files#r29350629>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/111/files#r29396093>.
>